client/src/components/ReviewTile.js

```diff
@@ -12,11 +15,58 @@ const ReviewTile = (props) => {
     }
   })
 
+  const handleClick = async (event) => {
+    setShowVoteError(false)
+    let alreadyVoted = false
+    props.votes.forEach(vote => {
```

Comment:
If we had lots of votes this would be very costly when it comes to time to iterate over all votes and determine if the user has already voted.
One option would be to check this on the back end and send an error message if they have already voted (**ideal**). Handling this on the back end makes it easier if/when we want to allow users to change/ delete their vote because we would already be in the backend environment.
Another option would be to use a speedier method like `find` rather than `forEach`, because `find` will return the first match, rather than continuing to iterate over all votes.